mysql/assets/recommended_monitors/mysql_replication_slave.json

```diff
@@ -0,0 +1,25 @@
+{
+    "name": "[MySQL] Replication slave not running",
+    "type": "query alert",
+    "query": "sum(last_1m):avg:mysql.replication.seconds_behind_master{*} by {host} < 0.99",
```

Comment:
using the sum time aggregator here is not a good idea. It would sum all the data points. Let's say you have 5 data points in the last minute: 0.4 - 0.4 - 0.4 - 0.5 - 0.6. The end result would be 2.3.

Also what is this 0.99 threshold?

Reply:
max would be a better idea for the threshold

Reply:
+ check the threshold